Added NGINXaaS for Azure doc on how to change from basic to standard plan (#1359)
nginx/documentation: content/nginxaas-azure/changelog/changelog.md

- {{% icon-info %}} **Retirement of Standard Plan**

>    The `Standard` plan for NGINXaaS for Azure has been retired, and you can no longer use it to create new deployments. If you have a deployment running on the `Standard` plan, consider [migrating]({{< ref "/nginxaas-azure/change-plan/migrate-from-standard.md">}}) it to the [`Standard V2 plan`]({{< ref "/nginxaas-azure/billing/overview.md#standard-v2-plan" >}}) to access new features such as F5 WAF for NGINX and additional listen ports. Plan migration does not incur downtime.

Reviewer: ```suggestion
   The `Standard` plan for NGINXaaS for Azure has been retired and can no longer be used to create new deployments. If you have an existing deployment on the `Standard` plan, consider [migrating]({{< ref "/nginxaas-azure/change-plan/migrate-from-standard.md">}}) to the [`Standard V2 plan`]({{< ref "/nginxaas-azure/billing/overview.md#standard-v2-plan" >}}) to access new features such as F5 WAF for NGINX and additional listen ports. Migrating your plan does not cause downtime.
```

- {{< icon-feature >}} **In-place SKU Migration from Standard to Standard V2**

>    You can now migrate NGINXaaS for Azure from the Standard plan to the Standard V2 plan without redeploying. We recommend upgrading to the Standard V2 plan to access features like F5 WAF for NGINX and more listen ports. The Standard plan will be retired soon. For migration details, see [migrate from standard]({{< ref "/nginxaas-azure/change-plan/migrate-from-standard.md">}}).

Reviewer: ```suggestion
   You can now migrate NGINXaaS for Azure from the `Standard` plan to the `Standard V2` plan without redeploying. We recommend upgrading to the `Standard V2` plan to access features such as F5 WAF for NGINX and additional listen ports. The `Standard` plan will be retired soon. For migration steps, see [Migrate from Standard]({{< ref "/nginxaas-azure/change-plan/migrate-from-standard.md">}}).
```

I added tick marks around `Standard` and `Standard V2` to match line 131. If the ticks aren't needed, remove in both places.
